src/Spec/Transform.purs

```diff
@@ -215,28 +220,26 @@ type FormLens form e i o =
   --  , output :: APrism' (form InputField) o
```

Comment:
Do you have any ideas on these? I had some trouble getting the types to resolve with prisms.

Reply:
This would require `ATraversal`, which doesn't exist (https://github.com/purescript-contrib/purescript-profunctor-lenses/issues/26)